tests/integrations/virtual_environment/test_VirtualEnvironmentContext__bin_dir.py

```diff
@@ -0,0 +1,55 @@
+from pathlib import Path
+from unittest.mock import patch
+
+import pytest
+
+from briefcase.integrations.virtual_environment import VenvContext
+
+
+class TestVirtualEnvironmentContextBinDir:
+    """Tests for VenvContext.bin_dir property."""
+
+    def test_bin_dir_unix(self, dummy_tools, tmp_path):
+        """Test bin_dir returns 'bin' directory on Unix systems."""
+        venv_path = tmp_path / "test_venv"
+        context = VenvContext(dummy_tools, venv_path)
+
+        with patch("os.name", "posix"):
```

Comment:
We tend towards using `sys.platform` rather than `os.name` as a platform check. I know the you've used this check in the implementation; I kind of let that slide on the basis that it was only a couple of uses - but if it's going to be needed more extensively, we should be consistent with the rest of Briefcase.